packages/appstore/src/components/options-multipliers-listing/index.tsx

```diff
@@ -27,13 +27,13 @@ const OptionsAndMultipliersListing = () => {
     const cr_demo = content_flag === ContentFlag.CR_DEMO;
 
     const OptionsTitle = () => {
-        if ((low_risk_cr_non_eu || high_risk_cr || cr_demo) && !isMobile()) {
+        if ((low_risk_cr_non_eu || high_risk_cr || cr_demo) && !is_mobile) {
```

Comment:
@arshad-rao-deriv Just a suggestion, you can ignore it if you want. In my opinion would be better to use early return here like this:

```
const OptionsTitle = () => {
   if (is_mobile) return null;
   if ((low_risk_cr_non_eu || high_risk_cr || cr_demo) { return ... };
   if ((low_risk_cr_non_eu || high_risk_cr || cr_demo) { return ... };
   if (low_risk_cr_eu || is_eu) { return ... };
   return null;
}
```

Reply:
Updated.